Workbooks/Communication Services/Insights_Beta/Insights_Beta.workbook

```diff
@@ -1,3560 +1,4679 @@
-                                "representation": "info",
-                                "text": "{0}{1}"
-                              }
-                            ]
-                          }
-                        },
-                        {
-                          "columnMatch": "OperationDate",
-                          "formatter": 5
-                        },
-                        {
-                          "columnMatch": "OperationTime",
-                          "formatter": 5
-                        }
-                      ],
-                      "hierarchySettings": {
-                        "treeType": 1,
-                        "groupBy": [
-                          "OperationDate"
-                        ],
-                        "expandTopLevel": true,
-                        "finalBy": "OperationTime"
-                      }
-                    },
-                    "sortBy": []
-                  },
-                  "conditionalVisibility": {
-                    "parameterName": "sms_version",
-                    "comparison": "isNotEqualTo"
-                  },
-                  "name": "sms-by-version"
-                }
-              ]
-            },
-            "customWidth": "50",
-            "name": "sms-operation-versions-group"
-          },
-          {
-            "type": 12,
-            "content": {
-              "version": "NotebookGroup/1.0",
-              "groupType": "editable",
-              "title": "SMS Method",
-              "items": [
-                {
-                  "type": 3,
-                  "content": {
-                    "version": "KqlItem/1.0",
-                    "query": "ACSSMSIncomingOperations\r\n| where TimeGenerated {time_range:query}\r\n| summarize count() by Method\r\n| render piechart",
-                    "size": 0,
-                    "noDataMessage": "No SMS operations found in the specified time range",
-                    "exportMultipleValues": true,
-                    "exportedParameters": [
-                      {
-                        "fieldName": "series",
-                        "parameterName": "sms_method",
-                        "parameterType": 1
-                      }
-                    ],
-                    "queryType": 0,
-                    "resourceType": "microsoft.communication/communicationservices"
-                  },
-                  "name": "sms-operation-names"
-                },
-                {
-                  "type": 3,
-                  "content": {
-                    "version": "KqlItem/1.0",
-                    "query": "let SIPCodesSMS =  datatable(ResultSignature: string, SIPCode: string)\r\n[\r\n    \"0\", \"Success\",\r\n    \"100\", \"Trying\",\r\n    \"180\", \"Ringing\",\r\n    \"181\", \"Call Is Being Forwarded\",\r\n    \"182\", \"Queued\",\r\n    \"183\", \"Session Progress\",\r\n    \"199\", \"Early Dialog Terminated\",\r\n    \"200\", \
```

Comment:
I don't think a bin of 0.5 makes sense for recipientCount since we cannot have an "halfRecipient" (likely just a count by ToRecipientsCount would work)

Also in the case an email have 10 ccRecipient, this visualization would be misleading ??

A Count of recipients should be for the Group ( To + CC + Bcc ) ?